Bio/PDB/QCPSuperimposer/__init__.py

```diff
@@ -87,7 +87,7 @@ def set(self, reference_coords, coords):
         self.coords = coords
         n = reference_coords.shape
         m = coords.shape
-        if n != m or not(n[1] == m[1] == 3):
+        if n != m or not n[1] == m[1] == 3:
```

Comment:
Is this definitely equivalent? The comparison of three terms is in itself perhaps not the best style...